gson/src/main/java/com/google/gson/Gson.java

```diff
@@ -548,7 +548,7 @@ static void checkValidFloatingPoint(double value) {
    *  read or written.
    * @param skipPast The type adapter factory that needs to be skipped while searching for
    *   a matching type adapter. In most cases, you should just pass <i>this</i> (the type adapter
-   *   factory from where {@link #getDelegateAdapter} method is being invoked).
+   *   factory from where {@code getDelegateAdapter} method is being invoked).
```

Comment:
Changed this to `code` because it refers to the current method itself, so having a `link` is redundant and might confuse readers (making them think the link refers to a different method).